smoketests/pages/android_feedback_form.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from pages.feedback import FeedbackPage
+
+
+class AndroidFeedbackFormPage(FeedbackPage):
+    _page_title = 'Firefox Feedback :: Firefox Input'
+
+    def go_to_feedback_page(self,
+                            version="",
+                            channel="",
+                            last_visited="",
+                            in_device=False):
```

Comment:
This is a very odd kwarg style, you should probably stick to keeping them on one, maybe two lines.

Also, you're using double quotes for some of the strings here when you should be using single quotes.